app/views/govuk_publishing_components/components/docs/image_card.yml

```
@@ -204,3 +205,12 @@ examples:
         }
       ]
       extra_links_no_indent: true
+  with_lang:
+    description: |
+      Pass through an appropriate `lang` to set a HTML lang attribute for the component.
```

Do you think it's worth adding what an appropriate `lang` attribute should be?

```suggestion
      Pass through an appropriate `lang` to set a HTML lang attribute for the component.
      
      The `lang` attribute **must** be set to a [valid BCP47 string](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/lang#Language_tag_syntax). A valid code can be the two or three letter language code - for example, English is `en` or `eng`, Korean is `ko` or `kor` - but if in doubt please check.
```

This was pinched from https://github.com/alphagov/govuk_publishing_components/commit/bfd89eef898d5c67b85dea1b6955a37288c47916#diff-745eea565982ff38d9c748f6a3c85e9eR31.